functions/dpeCronExpiredMediaChecker/index.js

```diff
@@ -0,0 +1,46 @@
+const { updateTranscription } = require("../sttChecker");
+
+const isDeletableContentType = (contentType) => {
+  return contentType ? contentType.includes("video") || contentType.includes("audio") : false;
+}
+
+const dpeCronExpiredMediaChecker = (bucket, admin) => {
```

Comment:
Since this function is calling an async function (updateTranscription) it should probably be declared async too.